SmartDeviceLink/SDLRadioControlData.h

```diff
@@ -0,0 +1,93 @@
+//
+//  SDLRadioControlData.h
+//
+
+#import "SDLRPCMessage.h"
+#import "SDLRadioBand.h"
+#import "SDLRadioState.h"
+
+@class SDLRdsData;
+
+/**
+ * Include information (both read-only and changeable data) about a remote control radio module.
+ */
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLRadioControlData : SDLRPCStruct
+
+- (instancetype)initWithFrequencyInteger:(nullable NSNumber<SDLInt> *)frequencyInteger frequencyFraction:(nullable NSNumber<SDLInt> *)frequencyFraction band:(nullable SDLRadioBand)band rdsData:(nullable SDLRdsData *)rdsData availableHDs:(nullable NSNumber<SDLInt> *)availableHDs hdChannel:(nullable NSNumber<SDLInt> *)hdChannel signalStrength:(nullable NSNumber<SDLInt> *)signalStrength signalChangeThreshold:(nullable NSNumber<SDLInt> *)signalChangeThreshold radioEnable:(nullable NSNumber<SDLBool> *)radioEnable state:(nullable SDLRadioState)state;
```

Comment:
This initializer should only contain properties such that are actually currently settable by developers (e.g. `rdsData` is readonly).